scripts/validate_phase6.py

```
@@ -0,0 +1,765 @@
+EXTRACTED_DIR = DATA_DIR / "extracted-data"
+CONSOLIDATED_DIR = EXTRACTED_DIR / "consolidated"
+OUTPUT_DIR = Path(os.environ.get("OUTPUT_DIR", Path(__file__).resolve().parent))
+
+# ── Load all data sources ──────────────────────────────────────────────────
+
+
+def load_json(path, label):
+    try:
+        with open(path, encoding="utf-8") as f:
+            return json.load(f)
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)